api/v1alpha1/batchrelease_plan_types.go

```diff
@@ -54,6 +54,11 @@ type ReleasePlan struct {
 	// only support for canary deployment
 	// +optional
 	PatchPodTemplateMetadata *PatchPodTemplateMetadata `json:"patchPodTemplateMetadata,omitempty"`
+	// RollingStyle can be "Canary", "Partiton" or "BlueGreen"
+	RollingStyle RollingStyleType `json:"rollingStyle,omitempty"`
+	// When user verifies that the canary version is ready, we will remove the canary deployment and release the deployment workload-demo in full.
+	// Current only support k8s native deployment
```

Comment:
plz comment the relationship with rollingStyle and enableExtraWorkloadForCanary

Reply:
done